src/index.js

```diff
@@ -14,8 +14,10 @@ import { WebSocketLink } from 'apollo-link-ws';
 import { getMainDefinition } from 'apollo-utilities';
 import { ApolloLink } from 'apollo-link';
 
-const GRAPHQL_ENDPOINT = '';
-const SUBSCRIPTIONS_ENDPOINT = '';
+import constants from './constants';
+
+const GRAPHQL_ENDPOINT = 'https://api.graph.cool/simple/v1/cjmqr4q4w57fr01849g8nega7';
```

Comment:
you probably don't want to check these in

Reply:
oh nooooo! what a stupid mistake, it's already to late

Reply:
It happens - make sure you delete that project on Graph.cool though (or at least don't use it for something important) :)